pyhanabi.cc

```diff
@@ -803,7 +803,11 @@ void NewObservationEncoder(pyhanabi_observation_encoder_t* encoder,
   switch (otype) {
     case hanabi_learning_env::ObservationEncoder::Type::kCanonical:
       encoder->encoder = static_cast<hanabi_learning_env::ObservationEncoder*>(
-          new hanabi_learning_env::CanonicalObservationEncoder(hanabi_game));
+          new hanabi_learning_env::CanonicalObservationEncoder(hanabi_game, false));
```

Comment:
this file should remain unchanged once we fix previous comments

Reply:
we are not using `pyhanabi.cc`. instead, we will use the derived `CanonicalObservationEncoder` directly in our c++ code. we should keep the changes minimal.